### ISSUE TITLE
Merge best practices as "special sections" with rule book

### ISSUE THREAD

**Issue by gryphonshafer** [gryphonshafer/Quizzing-Rule-Book#55]
We have the ability to include arbitrary "special sections" in the rule book that can be filtered during build. An example is "Logic", which contains English-Script that few will want included and which won't be included in standard builds.

I think it'll be a good idea to consider merging best practices into the rule book in a series of "Best Practice" special sections.

**Comment by kclimenhaga** [comment]
Another possibility would be to have "Best Practices" as endnotes. If we end up with a lot of best practices, it might be too wordy to have them inline, but I agree that a separate document would likely end up being ignored.

**Comment by jswingle** [comment]
Agreed with Kristen - the whole point of BPs is that we really want people to read them, even if they're not something that's challengeable. Making them as visible as possible is a good thing. Perhaps we could push for a rulebook render that is most useful for captains/coaches, and another render which emphasizes the BPs that is pushed on officials and question writers/editors?

**Comment by jswingle** [comment]
@gryphonshafer I think this is the kind of issue which the Rules Committee doesn't even need to vote on. It's just a tech implementation and the work would need to be done on your end. The RC should vote on what the Best Practices themselves are.

**Comment by gryphonshafer** [comment]
To accomplish this, there's a technical component (though extremely simple) and a more time-consuming "actually migrate the textual content" task.

For the technical component, if you want best practices to be integrated "special sections" in the document, then you only need to add or alter [the `builds` section of the YAML configuration file](https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/2a2ff6512de72ba364d921930d2d267996be1485/config/app.yaml#L36-L58). If you want the best practices to be a single section at the end, you can accomplish this the same way.

The "actually migrate textual content" would be something the RC should do.

**Comment by jswingle** [comment]
I think the right next step would be for the RC to take a vote on how they would like this content included, and then we can figure out the details from there. I would be in favor of the endnote formatting, for the reasons @kclimenhaga has already stated. A separate document would get ignored, but I also want a clear delineation between what is actually a challengeable/protestable rule and what is simply expected and usual.

**Comment by jswingle** [comment]
@gryphonshafer @josiah-leinbach @kclimenhaga 

I think we should continue to consider this issue, but I would like to more or less finalize a Best Practices document BEFORE we consider its implementation. That is a long way off, so I've assigned this to the "Backburner" tag for now. Agree/disagree?
